astropy/io/misc/asdf/tags/helpers.py

```diff
@@ -2,20 +2,7 @@
 # -*- coding: utf-8 -*-
 import numpy as np
 
+from astropy.coordinates.tests.helper import skycoord_equal
 
-__all__ = []
 
-
-def skycoord_equal(sc1, sc2):
```

Comment:
Since you have localized the deprecation warnings, do we also have to formally deprecate this function here?

Reply:
Oops, yes, thanks